examples/load-tester/load/utils/utils.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	pb "github.com/googleforgames/open-saves/api"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+func CreateTempStore(ctx context.Context, conn *grpc.ClientConn) (*pb.Store, error) {
```

Comment:
add export comments here. aside, why is this a temp store vs regular store?

Reply:
Because it's specifically for load testing and gets deleted after each run.